docs/package.json

```diff
@@ -82,8 +83,34 @@
   },
   "devDependencies": {
     "netlify-cli": "^2.59.0",
+    "remark-cli": "^8.0.1",
+    "remark-lint": "^7.0.1",
+    "remark-lint-no-dead-urls": "^1.1.0",
+    "remark-validate-links": "^10.0.2",
     "toml": "^3.0.0"
   },
+  "remarkConfig": {
+    "plugins": [
+      [
+        "remark-validate-links",
+        {
+          "prefix": "/docs/",
+          "lines": false
+        }
+      ],
+      "remark-lint",
+      [
+        "remark-lint-no-dead-urls",
+        {
+          "skipUrlPatterns": [
+            "^https://github\\.com/rasahq/rasa/issues/[0-9]+$",
+            "\\.prototyping\\.rasa\\.com",
+            "^https://github\\.com/mit-nlp/MITIE/releases/download/v0\\.4/MITIE-models-v0\\.2\\.tar\\.bz2$"
```

Comment:
This times out. I tried to use HEAD method but I got a 403. So I think we need to ignore it